src/clp_logging/decoder.py

```diff
@@ -38,8 +38,7 @@ def decode_int(token: bytes) -> Tuple[int, str]:
     @staticmethod
     def decode_float(token: bytes) -> Tuple[float, str]:
         """
-        See `CLPEncoder.encode_float` for detailed breakdown of CLP float
-        encoding.
+        See `encode_float` for detailed breakdown of CLP float encoding.
```

Comment:
I know you're going to remove the decoder for the ffi version, but for this PR can we say to see the file in the ffi repo that has this information.